apps/tester-federation/src/host/screens/MiniAppScreen.tsx

```diff
@@ -1,10 +1,7 @@
 import React from 'react';
-import { Federated } from '@callstack/repack/client';
 import { ActivityIndicator, StyleSheet, View } from 'react-native';
 
-const MiniAppNavigator = React.lazy(() =>
-  Federated.importModule('MiniApp', './MiniAppNavigator')
-);
+const MiniAppNavigator = React.lazy(() => import('MiniApp/MiniAppNavigator'));
```

Comment:
Yusss good riddance custom loader in userland! This is what I was hoping to see.